### PR TITLE
Data Federation Tool for Phnmnl stack

### PR DESCRIPTION
**Pull request by carmat88**
Tjena @andersla ! As we did for the logmon tools, please in this PR you can find all the changes in order to provision the phnmnl cluster with a data federation tool (namely Owncloud).

Any comments and/or changes, please do not hesitate to go for it!

Cheers.

### CODE REVIEW

File: bin/phenomenal-post-install.sh [hunk 1]
```diff
@@ -30,12 +30,27 @@ else
...
+  if [[ "$logmon" == true ]]; then
+    echo -e "\nLogging and Monitoring Tools:\n"
+    echo "http://kibana.$domain"
+    echo "http://prometheus.$domain"
+    echo "http://grafana.$domain"
+  fi
+  if [[ "$data_federation" == true ]]; then
+    echo -e "\nData Federation Tool:\n"
+    echo "http://owncloud.$domain"
+  fi
```
Comment (carmat88) [inline bin/phenomenal-post-install.sh]: Here I took the liberty of printing out optional service's URL if and only if their flags are equal to "true"

File: templates/config.tfvars.aws-template [hunk 2]
```diff
@@ -73,14 +73,15 @@ provision = {
...
-      "logmon_include"        = "false"
+      "username_all_services"   = "admin"
+      "password_all_services"   = "password"
+      "galaxy_admin_email"      = "yoourname@bla.bla.com"
+      "galaxy_include"          = "true"
+      "jupyter_include"         = "true"
+      "luigi_include"           = "true"
+      "dashboard_include"       = "false"
+      "logmon_include"          = "false"
+      "datafed_include"         = "false"
```
Comment (carmat88) [inline templates/config.tfvars.aws-template]: added the `"datafed_include"` label for all config.tfvars* templates.

File: templates/config.tfvars.azure-template [hunk 3]
```diff
@@ -19,7 +19,7 @@ node_vm_size = "Standard_DS2_v2"
 # Gluster configuration
 glusternode_count = "1"
 glusternode_vm_size = "Standard_DS2_v2"
-glusternode_extra_disk_size = "200" # Size in GB
+glusternode_extra_disk_size = "250" # Size in GB
```
Comment (carmat88) [inline templates/config.tfvars.azure-template]: Increased `glusternode_extra_disk_size` to 250 GB in order to allow all PVC to be allocated correctly when all services are enabled at once.

File: bin/phenomenal-post-install.sh (hunk 1 above)
Comment (andersla) [reply bin/phenomenal-post-install.sh]: Thanks! Great!

**Review by andersla — APPROVED** [review]
Looks great! Great job!

File: bin/phenomenal-post-install.sh (hunk 1 above)
Comment (andersla) [reply bin/phenomenal-post-install.sh]: You could change the post_install check to use same style of reading params as in kn-apply, e.g.

# Read config file into variable as as json
kn_config=$(json2hcl -reverse <config.tfvars)

# Set variables from json-config
galaxy_include=$(jq -r '.galaxy_include' <<<"$kn_config")
luigi_include=$(jq -r '.luigi_include' <<<"$kn_config")
dashboard_include=$(jq -r '.dashboard_include' <<<"$kn_config")

and so forth...

**Comment by andersla** [comment]
Minor suggestions that are very optional:)
You could change datafed_include to datafederation_include

**Comment by andersla** [comment]
Beautiful! You are a star:) Merge when you are ready!

**Pull request merged**